HuduAPI/Private/Invoke-HuduRequest.ps1

```diff
@@ -61,6 +61,13 @@ function Invoke-HuduRequest {
         'x-api-key' = (New-Object PSCredential 'user', $HuduAPIKey).GetNetworkCredential().Password;
     }
 
+    if (($null -ne $Int_HuduCustomHeaders) -and ($Int_HuduCustomHeaders.Count -gt 0)) {
```

Comment:
I would change the if statement to:
```suggestion
    if (($Script:Int_HuduCustomHeaders | Measure-Object).count -gt 0){
```

That makes sure you read it from the script scope and also makes the count more reliable.